fabtests/scripts/runfabtests.sh

```diff
@@ -596,15 +594,11 @@ function usage {
 	errcho -e " -vvv\tprint output of failing/notrun/passing"
 	errcho -e " -t\ttest set(s): all,quick,unit,functional,standard,short,complex (default quick)"
 	errcho -e " -e\texclude tests: comma delimited list of test names /
-			 regex patterns (with -R) e.g. \"dgram,rma.*write\""
+			 regex patterns e.g. \"dgram,rma.*write\""
 	errcho -e " -E\texport provided variable name and value to ssh client and server processes.
 			 options must of of the form '-E var=value'"
 	errcho -e " -f\texclude tests file: File containing list of test names /
-			 regex patterns (with -R) to exclude (one per line)"
-	errcho -e " -R\tTreat test exclusions as regex patterns"
-	errcho -e "   \tNote: the test names / patterns for -e and -f options
-			would be matched with the list of test names defined
-			in this script. They don't have fi_ prefix"
```

Comment:
We should probably keep this help text regarding the file patterns.

Reply:
I removed this since exclusions are always regular expressions now, so the fi_ prefix specified or not doesn't make a difference.

Reply:
Hmm, the user could also specify a stricter regex (e.g. ""^fi_cm_data"), that would only match tests that have correct prefix.

Reply:
I didn't realize that including the fi_ prefix didn't work.  I'll update the script to use the full binary name, which I think will be more intuitive.